.eslintrc.js

```diff
@@ -34,5 +34,6 @@ module.exports = {
       },
     ],
     '@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
+    '@typescript-eslint/no-var-requires': 0,
```

Comment:
Please use strings instead of numbers here. Why do you need it?

Reply:
Because checker throws errors on conditional imports like [here](https://github.com/software-mansion/react-native-reanimated/blob/86e14e38b906ed316c50877316e42ac42012ace4/src/reanimated2/NativeReanimated.js). I have changed this to a warning right now.